docs/reference/nuspec.md

```diff
@@ -303,6 +303,9 @@ Explicit references are typically used for design-time only assemblies. When usi
 
 Similarly, explicit references can be used for unit test frameworks, such as XUnit, which needs its tools assemblies located next to the runtime assemblies, but does not need them included as project references.
 
+> [!Important]
+> The `<references>` element is only supported correctly by projects using NuGet packages with `packages.config`. In order to support projects using `PackageReference` (such as .NET Core projects), the reference assemblies should also be copied to the `ref/<TFM>/` directory in the package.
```

Comment:
A Q for my own personal understand. 
`references` is not supported in PR in any manner? Is it completely ignored or interpreted incorrectly?

Reply:
It's used incorrectly. It correctly only puts the ref assemblies in the "compile" section of the assets file. I think the "runtime" section should contain all the dlls in the `lib\tfm` directory, but only contains the ref assemblies, so at runtime there are assembly load errors.

Although I consider it a bug, the discussion was that if we fix it, it increases the compatibility matrix (package authors have to do this for nuget clients before version x,y, and do this other thing for nuget clients with a higher version).  Since it's easy for package authors to resolve and this is an uncommonly used feature, our guidance will be for package authors to change their package to provide maximum compatibility.

Reply:
Makes sense.
Thanks.